docs/includes/giza.log

```diff
@@ -0,0 +1,21 @@
+WARNING:giza.content.changelog.tasks:changelog generation is not configured.
```

Comment:
It *is* in .gitignore, I think this one was generated before I added it to the ignore -- I thought I'd got them all, but apparently not. The .gitignore for the docs is also in the root of the mongodb/docs-php-library folder, so we should add the following to the ignore for this repo:

```
build/*
.DS_Store
build
*~
*giza.log
```